internal/pkg/service/common/etcdop/iterator/option.go

```diff
@@ -11,12 +11,12 @@ type Option func(c *config)
 
 type config struct {
 	prefix   string
-	serde    serde.Serde
+	serde    *serde.Serde
```

Comment:
Pre raw aj T iterator pouzijem ten isty config raw bude mat `serde = nil`, teda to musi byt pointer.